bucket/vesta.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "3.5.7",
+    "description": "VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies.",
+    "homepage": "https://jp-minerals.org/vesta/en/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://jp-minerals.org/vesta/en/download.html#:~:text=WWW%2DMINCRYST-,License%20agreement,-VESTA%20LICENSE%0AVersion"
```

```suggestion
        "url": "https://jp-minerals.org/vesta/en/download.html"
```

CI treats it as an invalid URL.

```
  [^] Line: D:\a\Extras\Extras\my_bucket\bucket\vesta.json:5:16
          [^] Path: http://scoop.sh/draft/schema#/definitions/licenseIdentifiers/Type
        [*] Error: String 'https://jp-minerals.org/vesta/en/download.html#:~:text=WWW%2DMINCRYST-,License%20agreement,-VESTA%20LICENSE%0AVersion' does not validate against format 'uri'.
```